tests/Behat/Mock/EventListener/FrankMartinShippingExportEventListener.php

```diff
@@ -12,21 +12,21 @@
 
 use BitBag\SyliusShippingExportPlugin\Entity\ShippingExportInterface;
 use Doctrine\Persistence\ObjectManager;
-use Sylius\Bundle\CoreBundle\Provider\FlashBagProvider;
+use Sylius\Bundle\CoreBundle\Provider\requestStackProvider;
```

Comment:
This won't work as it class name starts from lower case

Reply:
Thank You for pointing that out, it is fixed now.